src/private/WRLDBuildingHighlight.mm

```diff
@@ -0,0 +1,72 @@
+#import "WRLDBuildingHighlight.h"
+#import "WRLDBuildingHighlight+Private.h"
+#import "WRLDBuildingHighlightOptions+Private.h"
+#import "WRLDBuildingApiHelpers.h"
+#import "WRLDMathApiHelpers.h"
+
+#import "BuildingInformation.h"
+
+@interface WRLDBuildingHighlight ()
+
+@end
+
+@implementation WRLDBuildingHighlight
+{
+    int m_buildingHighlightId;
+    Eegeo::Api::EegeoBuildingsApi* m_buildingsApi;
+    UIColor* m_color;
```

Comment:
possibly use auto property here, similar to https://github.com/wrld3d/ios-api/blob/master/src/public/WRLDPolygon.h#L69